packages/slidev/node/export.ts

```diff
@@ -108,12 +110,18 @@ export async function exportNotes({
   base = '/',
   output = 'notes',
   timeout = 30000,
+  browserArg = 'chromium',
 }: ExportNotesOptions): Promise<string> {
-  if (!packageExists('playwright-chromium'))
-    throw new Error('The exporting for Slidev is powered by Playwright, please install it via `npm i -D playwright-chromium`')
+  const browserOption = ["chromium", "firefox", "webkit"].includes(browserArg) ? browserArg : null;
+  if (!browserOption) {
+    throw new Error("Available Playwright browsers are: chromium, firefox, webkit")
+  }
+  const browserPackage = `playwright-${browserOption}`;
+  if (!packageExists(browserPackage))
+    throw new Error('The exporting for Slidev is powered by Playwright, please install it via `npm i -D ' + browserPackage + '`')
 
-  const { chromium } = await import('playwright-chromium')
-  const browser = await chromium.launch()
+  const { playwright } = await import(browserPackage)
```

Comment:
Does this works fine?

I think the following seems better. Make bundler more friendly knows which is imported here, instead of a dynamic string, even if you have checked before.

```js
function loadBrowser(browserPackage) {
  if (browserPackage == 'chromium') {
    return import('playwright-chromium')
  } else ...
}
```